src/index.ts

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env node
+import * as yargs from "yargs";
+import * as AWS from "aws-sdk";
+
+// set the default region
+AWS.config.update({ region: "ap-southeast-2" });
```

Comment:
If we're going to be distributing this as an open source tool, should we let the AWS SDK just pick up on the `AWS_REGION` environment variable?